html_reference.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html lang='en'>
+	<head>
+		<link rel="stylesheet" href="css_stylesheet.css">
+		<script src="https://kit.fontawesome.com/c90803ff49.js" crossorigin="anonymous"></script>
+		<title>Milo Girardet</title>
+		<meta name="viewport" content="width=device-width, initial-scale=1">
+	</head>
+	<body onTouchStart="">
+	<header class="contactDetails">
+		<div class="container">
+			<div class="buttonsSentLeft">
+				<ul class="contactAnchors">
+					<li><a href="#" class="contactAnchorsStyling">milogirardet@gmail.com</a></li>
+					<li><a href="#" class="contactAnchorsStyling">07519645711</a></li>
```

Comment:
You can do the same here as with the email link but using a 'tel' link. So any users on mobile who click it will be taken to their phone with your number dialled in.

Reply:
Cool. Added, thank you!